editor/scene_tree_editor.cpp

```diff
@@ -673,6 +673,7 @@ void SceneTreeEditor::_renamed() {
 	}
 
 	String new_name = which->get_text(0);
+        String pristine_new_name = new_name;
```

Comment:
You should use tabs for indentation, not spaces.

Reply:
Will do.